test/serializer/pure-functions/AbstractPropertyDelete.js

```diff
@@ -0,0 +1,24 @@
+// additional functions
+// abstract effects
+
+let obj1 = global.__abstract ? __abstract('object', '({get foo() { return "bar"; }})') : {get foo() { return "bar"; }};
+let obj2 = global.__abstract ? __abstract('object', '({foo:{bar:"baz"}})') : {foo:{bar:"baz"}};
+if (global.__makeSimple) {
+  __makeSimple(obj2);
+}
+
+function additional1() {
+  delete obj1.yar;
+  return obj1.foo;
+}
+
+function additional2() {
+  delete obj2.bar;
```

Comment:
Neither of `delete`d properties exist.

For this reason, even if I replace the `createTemporalFromBuildFunction` inner callback return value with something like `return t.stringLiteral('hi');`, this test still passes.

Let's add another test where we delete an unknown property but also ensure it has been deleted.